lib/rubocop/cop/mixin/frozen_string_literal.rb

```diff
@@ -32,11 +32,12 @@ def uninterpolated_string?(node)
         node.str_type? || (node.dstr_type? && node.each_descendant(:begin).none?)
       end
 
-      def frozen_heredoc?(node)
+      def uninterpolated_heredoc?(node)
```

Comment:
This name seems more accurate. I left an alias since this is technically public api but after a github code search I doubt anyone actually uses this right now